docs/development.md

```diff
@@ -128,8 +128,9 @@ You need a student account which can be created by instructors.
 
 ## Testing
 
-TEAMMATES automated testing requires Firefox or Chrome (works on Windows and OS X).
-It is recommended to use Firefox 46.0 as this is the browser used in CI build (Travis/AppVeyor).
+* TEAMMATES automated testing requires Firefox or Chrome (works on Windows and OS X).
+  * It is recommended to use Firefox 46.0 as this is the browser used in CI build (Travis/AppVeyor).
+  * View and edit `test.selenium.browser` in `src/test/resources/test.properties` to change browser for testing (default is `firefox`)
```

Comment:
Missing period.

Reply:
Cool, thank you for your meticulousness!